### PR TITLE
Add GELF out plugin + move alias grok filter in input filter section

### PR DESCRIPTION
**Pull request by fbalicchia**

### CODE REVIEW

File: lib/plugins/output/gelfout.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+'use strict'
+var graygelf = require('graygelf')
+
+
+function OutputGELF (config, eventEmitter) {
+  this.config = config
+  this.config.port = config.port || 12100
+  this.config.address = config.address || '0.0.0.0'
```
Comment (otisg) [inline lib/plugins/output/gelfout.js]: Would "host" be better, more specific, than "address"?

File: bin/logagent.js [hunk 2]
```diff
@@ -35,8 +35,8 @@ var moduleAlias = {
   'postgresql-query': '../lib/plugins/input/postgresql.js',
   'input-tcp': '../lib/plugins/input/tcp.js',
   'input-kafka': 'logagent-input-kafka',
-  'grok': 'logagent-input-filter-grok',
   'input-influxdb-http': '../lib/plugins/input/influxHttp.js',
+  'input-zeromq': '../lib/plugins/input/index.js',
```
Comment (otisg) [inline bin/logagent.js]: Did you really mean to add this now?

Comment (otisg) [reply bin/logagent.js]: Aha, it looks like you removed it in a later commit.

File: lib/plugins/input/gelf.js [hunk 3]
```diff
@@ -30,7 +30,7 @@ InputGELF.prototype.createServer = function () {
   this.server = gelfserver();
   this.server.on('message', function (gelf) {
     //At the moment gelf server doesn't expose rinfo
```
Comment (otisg) [inline lib/plugins/input/gelf.js]: What is rinfo?  Is that a typo or does it really refer to something?

Comment (fbalicchia) [reply lib/plugins/input/gelf.js]: Yes, the comment is not clear 👎 . With rinfo I mean remote info I've see now this [issue] (https://github.com/wavded/graygelf/issues/17). As suggest, we can retrieve it from raw message. I'll try it to improve the PR. thanks for note

**Comment by fbalicchia** [comment]
Hi, now is possible to know remote address opsss _host_, I'm always wrong it ;-),  and port

**Comment by otisg** [comment]
Thanks @fbalicchia!

**Pull request merged**